helmfile/getContext.sh

```diff
@@ -36,9 +36,9 @@ getValue()
     if [ -z $GITHUB ];
     then
       echo "Fetching Secret $VALUE"
-      export $VALUE=$(aws secretsmanager get-secret-value --secret-id $VALUE --query SecretString --output text)      
+      export $VALUE=$(aws secretsmanager get-secret-value --secret-id $VALUE --query SecretString --output text --region ca-central-1)      
     else
-      echo "$VALUE=$(aws secretsmanager get-secret-value --secret-id $VALUE --query SecretString --output text)" >> "$GITHUB_ENV"
+      echo "$VALUE=$(aws secretsmanager get-secret-value --secret-id $VALUE --query SecretString --output text --region ca-central-1)" >> "$GITHUB_ENV"
```

Comment:
This removes the requirement to have `AWS_REGION` set, which configuration is not automated within the devcontainer.